package.json

```diff
@@ -87,7 +87,7 @@
     "hosted-git-info": "^4.0.2",
     "html-entities": "^2.3.2",
     "jsdom": "^19.0.0",
-    "json-api-serializer": "^2.2.1",
+    "json-api-serializer": "^2.6.6",
```

Comment:
@miguelcobain for the same or "single purpose" PR, do you mind to focus on deps updates separately?

Reply:
I think this is fine to keep in 👍